providers/apis/polymarket/api_handler.go

```diff
@@ -21,24 +17,14 @@ const (
 	// Name is the name of the Polymarket provider.
 	Name = "polymarket_api"
 
-	host = "clob.polymarket.com"
 	// URL is the default base URL of the Polymarket CLOB API. It uses the midpoint endpoint with a given token ID.
-	URL = "https://clob.polymarket.com/midpoint?token_id=%s"
+	URL = "https://clob.polymarket.com/markets/%s"
 
 	// priceAdjustmentMax is the value the price gets set to in the event of price == 1.00.
```

Comment:
Can remove this comment since priceAdjustmentMax not relevant anymore

Reply:
updated [fix comment](https://github.com/skip-mev/slinky/pull/666/commits/3700ff25371334bc037c6e93ae8e4864bdf5ff87)